src/main/java/core/basesyntax/exeptions/FileWriteException.java

```
@@ -0,0 +1,15 @@
+package core.basesyntax.exeptions;
+import java.io.IOException;
+public class FileWriteException extends RuntimeException {
+    public FileWriteException(String message, IOException e) {
+        super(message);
```

This constructor discards the original cause; it calls `super(message)` while the caller (ReportExporterDaoImpl) passes the IOException as the second parameter. This violates the checklist guidance: "Don't ignore exceptions in try-catch blocks" and "Let's rethrow a RuntimeException with an **informative** message and exception object." Change the constructor to pass the cause to the superclass, e.g. `super(message, e);` so the original IOException is preserved.